src/components/admin/DeleteAdminModal/DeleteAdminModal.tsx

```diff
@@ -0,0 +1,47 @@
+import { removeWorldAdmin } from "api/world";
+
+import { UserWithId } from "types/id";
+
+import { useWorldAndSpaceByParams } from "hooks/spaces/useWorldAndSpaceByParams";
+
+import { Modal } from "components/molecules/Modal";
+
+import { Button } from "../Button";
+
+export interface DeleteAdminModalProps {
```

Comment:
If not used outside this file

```suggestion
interface DeleteAdminModalProps {
```